packages/zowe-explorer/src/webviews/src/zos-console/App.tsx

```diff
@@ -0,0 +1,92 @@
+import { Dropdown, Option, TextArea, TextField } from "@vscode/webview-ui-toolkit";
+import { VSCodeDropdown, VSCodeTextArea, VSCodeTextField } from "@vscode/webview-ui-toolkit/react";
+import { useEffect, useState } from "preact/hooks";
+
+declare const vscode: any;
+
+export function App() {
+  const [consoleContent, setConsoleContent] = useState("");
+  useEffect(() => {
+    window.addEventListener("message", (event) => {
```

Comment:
Sonar Cloud is not picking up the lines from 11-20 that is running the check to prevent users from sending data into webview outside of extension/webview context.